core/geometry/src/bspline/BSplineSurface.ts

```diff
@@ -143,7 +143,7 @@ export interface BSplineSurface3dQuery {
    */
   poleStepUV(select: UVSelect): number;
 
-  /*
+  /**

```

Comment:
Why are double asterisks getting changed to single asterisks in all these seemingly unrelated files?

Reply:
These changes are from master, I added back the second asterisk.

Reply:
What do you mean? Master has double asterisk. Is your editor doing something funky?
https://github.com/imodeljs/imodeljs/blob/9a466c589ecae481faeb0199adea9b17b729bfbb/core/geometry/src/bspline/BSplineSurface.ts#L140

Reply:
@SpencerWBarnes the "resolve" button is for the commenter, not the PR submitter. Have you determined what caused this?

Reply:
@pmconne the fix is from this line below the poleStepUv,

https://github.com/imodeljs/imodeljs/blob/master/core/geometry/src/bspline/BSplineSurface.ts#L146